awx/playbooks/project_update.yml

```diff
@@ -24,6 +25,8 @@
   gather_facts: false
   connection: local
   name: Update source tree if necessary
+  vars:
+    git_task_env:  # configure in settings
```

Comment:
Is this necessary? Can't we just give a default below like we do for `scm_accept_hostkey` etc?

```yaml
environment: "{{ git_task_env | default('') }}"
```

Reply:
If we did it that way, I believe the default would be `{}`, not `''`, and we could... but I like having it at the play level to declare where it comes from.